### DISCUSSION TITLE
lesson 7: brownie.exceptions.RPCConnectionError: Able to launch RPC client, but unable to connect.

### DISCUSSION THREAD

**Discussion by ti51120** [smartcontractkit/full-blockchain-solidity-course-py#496] [Q&A] [answered]
I am doing brownie test just like https://youtu.be/M576WGiDBdQ?t=23329, but it shows the following error. I do see requests in my alchemy dashboard... Can someone give me clues about where might be wrong? thanks!! 

![image](https://user-images.githubusercontent.com/52118731/144757150-ab8fd89a-765f-478d-a522-7797e7718444.png)

**Comment by cromewar** [comment, marked as answer]
Hello @ti51120 it seems is a problem with the connection to alchemy, please make sure you did the correct configuration on alchemy it should look something like this:

![image](https://user-images.githubusercontent.com/20199180/144763788-f15dbc64-bd5b-44e7-8b9e-3c8b9abc0010.png)

also try to delete the network from brownie and add it again, may be you made a typo add adding the network.

I hope this info might help, best regards Cromewar.

**Reply by ti51120** [reply]
GM @cromewar,
After I do it all over again, it connects successfully and is able to do brownie test normally. It seems like a typo somewhere. Thanks for your help!!

**Reply by cromewar** [reply]
I'm glad that tip helped you! I'll close this thread now. Please feel free to make any other questions.

**Reply by freddie71010** [reply]
this worked for me too! thanks @cromewar !

**Reply by cromewar** [reply]
You are welcome @freddie71010
